lib/experimental/Widgets/Content/EventsList/event.tsx

```diff
@@ -41,9 +41,9 @@ export const Event = forwardRef<HTMLDivElement, EventProps>(
           }
         />
         <div className="flex flex-col gap-0.5">
-          <div className="flex flex-row gap-1">
+          <div className="flex flex-row gap-1 font-medium">
             <p>{title}</p>
-            <p className="text-f1-foreground-secondary">{subtitle}</p>
```

Comment:
Why do they have the same color now? 🤔 The subtitle should be lighter as on figma

Reply:
I think I know where the confussion might come from. The subtitle is the bit between parenthesys. As you can see in the image, it's the same colour.

![image](https://github.com/user-attachments/assets/3d5447c4-a785-4886-a95b-a9db64394b54)

Reply:
The lighter bit would be this one 
![image](https://github.com/user-attachments/assets/1d04b27f-4d24-4797-ae95-2381fdc40bcc)

Reply:
Oooh true